apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/DocumentDetailActivity.java

```diff
@@ -34,6 +34,7 @@
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
```

Comment:
nit: sort imports

Reply:
It turns out I was wrong. Android Studio sorts `import java.util.concurrent.TimeUnit` after `import java.util.Map`.